Headers/AppLovinSDK.framework/Headers/ALSdkSettings.h

```diff
@@ -0,0 +1,112 @@
+ * `NSUserTrackingUsageDescription` string in your `Info.plist` file.
+ *
+ * This defaults to the value that you entered into your `Info.plist` file via `AppLovinConsentFlowInfo`⇒`AppLovinConsentFlowEnabled`.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/consent-flow#info.plist">MAX Integration Guide ⇒ iOS ⇒ Consent Flow ⇒ Info.plist</a>
+ */
+ // [PLP] "should be enabled" by whom?
+@property (nonatomic, assign, getter=isEnabled) BOOL enabled;
+
+/**
+ * URL for your company’s privacy policy. This is required in order to enable the consent flow.
+ *
+ * This defaults to the value that you entered into your `Info.plist` file via `AppLovinConsentFlowInfo`⇒`AppLovinConsentFlowPrivacyPolicy`.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/consent-flow#info.plist">MAX Integration Guide ⇒ iOS ⇒ Consent Flow ⇒ Info.plist</a>
+ */
+@property (nonatomic, copy, nullable) NSURL *privacyPolicyURL;
+
+/**
+ * URL for your company’s terms of service. This is optional; you can enable the consent flow with or without it.
+ *
+ * This defaults to the value that you entered into your `Info.plist` file via `AppLovinConsentFlowInfo`⇒`AppLovinConsentFlowTermsOfService`.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/consent-flow#info.plist">MAX Integration Guide ⇒ iOS ⇒ Consent Flow ⇒ Info.plist</a>
+ */
+@property (nonatomic, copy, nullable) NSURL *termsOfServiceURL;
+
+- (instancetype)init NS_UNAVAILABLE;
++ (instancetype)new NS_UNAVAILABLE;
+
+@end
+
+/**
+ * This class contains settings for the AppLovin SDK.
+ */
+@interface ALSdkSettings : NSObject
+
+/**
+ * Settings relating to the AppLovin consent flow.
+ */
+@property (nonatomic, strong, readonly) ALConsentFlowSettings *consentFlowSettings;
+
+/**
+ * A toggle for verbose logging for the SDK. This is set to `NO` by default. Set it to `NO` if you want the SDK to be silent (this is recommended for App Store
+ * submissions).
+ *
+ * If set to `YES` AppLovin messages will appear in the standard application log which is accessible via the console. All AppLovin log messages are prefixed
+ * with the `/AppLovinSdk: [AppLovinSdk]` tag.
+ *
+ * Verbose logging is <em>disabled</em> (`NO`) by default.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/advanced-settings#enable-verbose-logging">MAX Integration Guide ⇒ iOS ⇒ Advanced Settings ⇒ Enable Verbose Logging</a>
+ */
+@property (nonatomic, assign) BOOL isVerboseLogging;
+
+/**
+ * Whether to begin video ads in a muted state or not. Defaults to `NO` unless you change this in the dashboard.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/advanced-settings#mute-audio">MAX Integration Guide ⇒ iOS ⇒ Advanced Settings ⇒ Mute Audio</a>
+ */
+ // [PLP] {@link} to the part of the dashboard where you can change this?
```

Comment:
@applovinjosh - could you point David to it? I'm actually not even sure if we support this feautre in the pub dashboard anymore.